ocaml/parsing/parser.mly

```diff
@@ -4194,17 +4212,22 @@ with_type_binder:
 
 /* Polymorphic types */
 
-%inline typevar: (* : string with_loc * jkind_annotation option *)
+%inline typevar: (* : (position * position) * string * jkind_annotation option *)
```

Comment:
IIUC this comment should still say `string with_loc` rather than `string`, as I think `mkrhs` adds a location.